### PR TITLE
Update GOARCH options and flag in environment set

### PR DESCRIPTION
**Pull request by calbritt**
- Add build option for Apple Silicon 
- Change flag to uppercase to avoid duplicate -c flag

### CODE REVIEW

File: cmd/environment.go [hunk 1]
```diff
@@ -130,7 +130,7 @@ func newEnvironmentCreateCmd() *cobra.Command {
 	flags.StringVarP(&consensus, "consensus", "k", "raft", "consensus algorithm to use for the given protocol, raft or ibft for quorum, poa for geth")
 	flags.BoolVarP(&multiRegion, "multi-region", "R", false, "whether to enable multi region")
 	flags.IntVarP(&blockPeriod, "block-period", "P", 0, "block period in seconds")
-	flags.UintVarP(&chainID, "chain-id", "c", 0, "Chain ID")
+	flags.UintVarP(&chainID, "chain-id", "C", 0, "Chain ID")
```
Comment (vdamle) [inline cmd/environment.go:133]: Nice catch! I'm assuming there is going to be a follow-on update to the Terraform provider?

**Review by vdamle — APPROVED** [review]
leaving it approved, this PR would be worth a review from @peterbroadhurst as well, given the command line option fix.

File: cmd/environment.go (hunk 1 above)
Comment (calbritt) [reply cmd/environment.go:133]: Thanks! What would need to be updated in the Terraform provider?

**Comment by calbritt** [comment]
Closing with changes updated in later pr - https://github.com/kaleido-io/kaleido-sdk-go/pull/45

**Pull request closed without merging**